docs/extensions/services/interacting-with-files.md

```diff
@@ -0,0 +1,63 @@
+---
+description: Learn about the FilesService in Directus and how to utilize them when building extensions.
+contributors: Esther Agbaje
+---
+
+# Interacting With Files
+
+Directus exposes services that allow you to perform operations programmatically on your files.
+
+## FilesService
```

Comment:
I would remove this for consistency with ItemsService. I also think it's slightly redundant.